docs/MAINTAINING.md

```diff
@@ -0,0 +1,84 @@
+## Maintaining VTEX Toolbelt
+
+This document is for people working on VTEX as Toolbelt maintainers.
+
+### Forcing a new version
+
+Toolbelt has a way of enforcing users to use a minimum version choosed by us. This enforcement is done on the client side (can be disabled by an environment variable, check [EnvVariables](../src/lib/constants/EnvVariables.ts)) and on `vtex.builder-hub` - users using an older toolbelt version will not be allowed to link or publish. This feature is useful for enforcing users to update to the latest version, which can benefit users in many situations, like the following:
+
+- Bugs on toolbelt usage were fixed, providing a better experience.
```

Comment:
That's a good way to communicate why the minimumToolbeltVersion is important 

LGTM! :D